types/ember-data/test/store.ts

```diff
@@ -155,15 +155,22 @@ const tom = store
     });
 
 // GET /users?isAdmin=true
-const admins = store.query('user', { isAdmin: true });
-assertType<DS.AdapterPopulatedRecordArray<User> & DS.PromiseArray<User, Ember.ArrayProxy<User>>>(admins);
+const adminsQuery = store.query('user', { isAdmin: true });
+assertType<DS.PromiseArray<User, DS.AdapterPopulatedRecordArray<User>>>(adminsQuery);
+
+adminsQuery.then(function (admins) {
+    console.log(admins.get("length")); // 42
```

Comment:
It looks like the original tests came from the example at [`AdapterPopulatedRecordArray`](https://api.emberjs.com/ember-data/release/classes/AdapterPopulatedRecordArray), but I don't think the usage of..

```js
admins.then(function() {
  console.log(admins.get("length")); // 42
});
```

..is correct, as `admins` is already the resolved value from the query